cmd/horologium/main.go

```diff
@@ -227,3 +239,54 @@ func sync(prowJobClient ctrlruntimeclient.Client, cfg *config.Config, cr cronCli
 	}
 	return nil
 }
+
+func shouldTriggerFailedRun(j v1.ProwJob, p config.Periodic, now time.Time, logger *logrus.Entry, previousFound bool) bool {
+	if p.RetriggerFailed == nil {
+		return false
+	}
+	runCount := 0
+	if previousFound {
+		if !j.Complete() {
+			return false
+		}
+		countLabel, exists := j.Labels[kube.ReRunLabel]
+		if exists {
+			if count, err := strconv.Atoi(countLabel); err == nil {
+				runCount = count + 1
+			}
+		}
+	}
+	attempts := p.RetriggerFailed.Attempts
+	if attempts > maxRetryCount {
+		attempts = maxRetryCount
+	}
+	if runCount >= attempts {
```

Comment:
This does not seem correct. 

A vanilla-triggered job will have a label of `0`. So `runCount` will be `1` when we consider to retry it (L255). So when I configure `Attempts = 1`, I will expect a single retry but I will get none?